nuget/MANAGED_PACKAGE.md

```diff
@@ -0,0 +1,3 @@
+## About
+
+This package is a dependency of [Microsoft.ML.OnnxRuntimeGenAI](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntimeGenAI) and does not need to be installed directly.
```

Comment:
Since we have multiple packages, do we need to highlight that?

Reply:
Maybe?! I thought about it. What do you think?

Reply:
I think I'll leave as hopefully the instruction is enough to say the user does not have to depend on this package